### ISSUE TITLE
Video to demonstrate key features

### ISSUE THREAD

**Issue by maxdraki** [ExpediaGroup/insights-explorer#391]
Hey @baumandm - how are you? 
I've no idea what you're up to on this, but if you don't have the time to make a deployable container that I could use to demo, is there any chance you could record a quick video showing some of the key features?
Thank you!

**Comment by baumandm** [comment]
Hey Max, thanks for the reminder on both issues.  The biggest blocker on this is getting some compelling looking fake data to populate a demo instance with.  I spent some time attempting GPT text generation but didn't get good enough results.  We'll prioritize both and try to get something out soon.

**Comment by maxdraki** [comment]
> I spent some time attempting GPT text generation

Why am I not surprised :laughing: Nice idea though (I looked at using wikipedia content, but got lost in license fineprint!) Maybe just Lorem Ipsum?
LMK if there's anything I can help with from a distance

**Comment by baumandm** [comment]
Good idea to use lorem ipsum.  I've created this as a rough first step: https://github.com/baumandm/lorem-insight

**Comment by maxdraki** [comment]
Wow - that's very cool!! Nice work Dave


***@***.*** ***@***.***>
07803 009 004
